analyses/transcriptomic-dimension-reduction/ci-dimension-reduction-plots.sh

```diff
@@ -41,6 +41,16 @@ Rscript --vanilla scripts/run-dimension-reduction.R \
   --neighbors 2 \
   --skip_tsne
 
+# Run at tumor purity threshold
```

Comment:
Missing `--log2_transform` maybe? If only for completeness.

Reply:
Done in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1668/commits/4d6bd6f9e187df984e81222b73f238476e166b0a